parler_tts/modeling_parler_tts.py

```diff
@@ -3442,6 +3443,7 @@ def generate(
                 generation_config=generation_config,
                 synced_gpus=synced_gpus,
                 streamer=streamer,
+                logits_warper=None,
```

Comment:
You should keep the logits_warper, I'm not sure why you removed it!

Reply:
I didn't remove it! Originally, `logits_warper` wasn't being passed in, so this part of the code was failing. I believe when doing greedy search, `logits_warper=None` should be set. Please could you double check this!